src/openapi-vuepress-markdown.ts

```diff
@@ -17,18 +17,33 @@ async function main(): Promise<void> {
             '-e, --endpoints-prefix <endpoints-prefix>',
             'Endpoint file prefix, to avoid filename conflicts with resources. Defaults to blank (no prefix)',
         )
-        .parse()
+        .option(
+            '-et, --endpoints-template <endpoints-template>',
+            'Endpoints template file. Defaults to ./endpoints.md',
+        )
+        .option(
+            '-rt, --resource-template <resource-template>',
```

Comment:
``--endpoints-remplate`` has ``endpoints`` plural, but ``--resource-template`` is singular, I'd pick one or the other

Reply:
It is intentional.
Endpoints are parsed by tag. One endpoints file is generated for all endpoints of a given tag. This is to group them together and produces nice markdown headers.

Resources, on the other hand, are parsed individually and one markdown file per resource is generated.

Reply:
I see, fair